core/src/main/java/org/testcontainers/dockerclient/DockerMachineClientProviderStrategy.java

```diff
@@ -16,6 +16,9 @@
  */
 @Slf4j
 public class DockerMachineClientProviderStrategy extends DockerClientProviderStrategy {
+
+    public static final int PRIORITY = EnvironmentAndSystemPropertyClientProviderStrategy.PRIORITY - 10;
```

Comment:
Sure! Since we remember last used strategy, I think we can remove them now. Will do as a follow up